lib/editor/components/pattern/PatternStopContainer.js

```diff
@@ -111,20 +111,20 @@ class PatternStopContainer extends Component<Props, State> {
   render () {
     const {
       activePattern,
+      addStopToPattern,
```

Comment:
Something happens during the e2e test when adding a pattern stop to a route, the entire screen blanks out, and the only visible thing is the yellow notification popup (see screenshot).

![e2e-error-should_create pattern-2022-08-10T17-34-27](https://user-images.githubusercontent.com/56846598/184027154-8507f6f0-b1eb-4a4f-8dcd-1d3eb3d53d87.jpeg)

Reply:
It seems like this crash is only happening when running locally, but the exception one is happening on both localhost and the ci.

Reply:
The problem on the ci e2e tests seems to be it's expecting the project list to show up, but for some reason it's getting some specific project page instead...

Reply:
Going to resolve this for now as this test failure seems to be unrelated to react 17. I will block this until the e2e tests are dockerized.